vmm/src/vmm_config/instance_info.rs

```diff
@@ -101,6 +101,8 @@ pub enum StartMicrovmError {
     VcpuConfigure(vstate::Error),
     /// vCPUs were not configured.
     VcpusNotConfigured,
+    /// vCPUs were not created prior to starting them.
+    VcpusNotCreated,
```

Comment:
Is it possible to ever get this error? At least in current implementation, it seems `start_vcpus` is invoked right after `create_vcpus`, so they're always created before being started.

Reply:
You can get this error if `start_vcpus` is called before `create_vcpus` (programming error) but this misuse will also be caught by the ` assert_eq!(vcpus.len(),
            vcpu_count as usize,
            "The number of vCPU fds is corrupted!"
        );`
so I will only leave this assert and remove this error from this enum which indeed should only keep around user triggered errors.

Reply:
Removed the error. Take a look.